Source/Extensions/Font+Application.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Font+Application.swift
+//  GithubNotifications
+//
+//  Created by Chananchida F. on 9/8/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import SwiftUI
+
+extension Font {
+
+    static let headlineTitle: Font = { Font.system(size: 22, weight: .bold) }()
```

Comment:
I prefer if we use the `Dynamic Type Sizes` section in https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/ for font sizing and add modifier in declarative fashion `Text("Simple Swift Guide").font(.headline).bold().italic()`. Have to wait for other for this.

Reply:
Yes, we can use `Dynamic Type Sizes`

Reply:
@blyscuit I updated the code. Please help check 🙇‍♀️